### DISCUSSION TITLE
EddieCon 0.1 Speaker Resources

### DISCUSSION THREAD

**Discussion by naomi-lgbt** [EddieHubCommunity/support#2221] [Conference Notes]
Calling all EddieCon Speakers!

Share the resources relevant to the talks you gave today, so folks can continue their learning!

**Comment by naomi-lgbt** [comment]
Here's the code I used for the slash commands demonstration: https://github.com/nhcarrigan-archive/eddiecon-0.1-demo

**Comment by Sam-Varghese** [comment]
[Here](https://sam-varghese.github.io/EddieCon/) are my slides of the talk on the topic *Python For Everybody*

**Comment by kaiwalyakoparkar** [comment]
Here is my node-js-project template repo to fork/clone and use :) https://github.com/kaiwalyakoparkar/node-js-project-template
